app/helpers/image_encode_helper.rb

```diff
@@ -0,0 +1,33 @@
+module ImageEncodeHelper
+  def encodable_image_tag(source, options = {})
+    image_tag(encodable_image_source(source), options)
+  end
+  alias eimage_tag encodable_image_tag
```

Comment:
Just a future convenience. Any image that might ever make it into a PDF should use this helper.